package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@kilohealth/web-app-monitoring",
+  "version": "1.0.0-alpha.12",
+  "license": "MIT",
+  "author": "Kilo Health",
+  "main": "dist/index.js",
+  "bin": {
+    "web-app-monitoring__upload-sourcemaps": "dist/cli/sourcemap-uploader.sh"
+  },
+  "scripts": {
+    "build": "rimraf dist && tsc && npm run build:cli",
+    "build:cli": "cp -r ./src/cli ./dist/cli",
+    "prepare": "husky install",
+    "semantic-release": "semantic-release",
+    "test": "jest",
+    "upload:sourcemaps": "./dist/cli/sourcemap-uploader.sh"
+  },
+  "lint-staged": {
+    "package.json": [
+      "sort-package-json"
+    ],
+    "*.{json,md,yml}": [
+      "prettier --write"
+    ],
+    "*.{ts,tsx,js,jsx}": [
+      "prettier --write",
+      "eslint --max-warnings=0 --fix"
+    ]
+  },
+  "prettier": "@kilohealth/prettier-config",
+  "eslintConfig": {
+    "extends": "@kilohealth/eslint-config-node"
+  },
+  "dependencies": {
+    "@datadog/browser-logs": "^4.42.2",
+    "@datadog/datadog-ci": "^2.11.0",
+    "dd-trace": "^4.0.0",
+    "lodash": "^4.17.21",
```

Comment:
how about using simple [deepmerge](https://www.npmjs.com/package/deepmerge) or even custom merging utility instead of huge loadash dependency